common/src/main/java/com/aliyun/emr/rss/common/network/server/FileInfo.java

```diff
@@ -24,28 +24,37 @@
 import org.apache.hadoop.fs.FSDataOutputStream;
 
 public class FileInfo {
-  public final File file;
+  private final String filePath;
```

Comment:
Better to move this and FileManagedBuffers.java to com.aliyun.emr.rss.common.meta